src/lib.rs

```diff
@@ -162,6 +162,78 @@ impl Calc {
                 scope.var_store.insert(id.to_string(), val);
                 val
             }
+            BinaryOp::Equal => {
+                let op1 = self.stack_pop()?;
+                let op2 = self.stack_pop()?;
+                let stack_value;
+                if op1.is_same_type(&op2) {
+                    stack_value = StackValue::Boolean(op1.eq(&op2));
+                    self.stack.push(stack_value)
+                } else {
+                    return Err(InterpError::EvalError(
+                        format!(
+                            "Operand {} and Operand {} cannot be applied to Equal operation",
+                            op1, op2
+                        )
+                        .to_string(),
+                    ));
+                }
+                stack_value
+            }
+            BinaryOp::NotEqual => {
```

Comment:
Have you considered implementing `!=` in terms of `==`. Do the same computation, but negative the end result?

Reply:
Good point. Implemented :point_right: a34a7051380f080122a538d286b5729e1a101de0